core/src/test/java/tech/tablesaw/SortTest.java

```diff
@@ -99,7 +101,7 @@ public void testAscendingWithPlusSign() {
         compareTables(expectedResults, sortedTable);
     }
 
-    @Test(expected = ComparisonFailure.class)
+    @Ignore
```

Comment:
This test was always failing, which is what the `expected = ComparisonFailure` indicated. It's more standard to use `@Ignore` for disabled tests though. `expected` is intended to be used when you want to test that an `Exception` is thrown. It was failing because of https://github.com/jtablesaw/tablesaw/issues/354